### PR TITLE
56570 Add Separation Health Assessment to BDD claims

### LINKED ISSUE #56570: Content changes for BDD - SHA Part 1
## Issue Description
Users who are active service members submitting a 526 under the BDD (Benefits Delivery at Discharge) Program will need to also submit a Part A Self-Assessment form as of April 1 in order to get an exam before discharge from service.

This ticket is to implement content changes on the following pages of the form to inform/direct the veteran of the requirement to complete and upload the Part A Self-Assessment form along with the claim to not hold-up further processing:
- Content changes to BDD intro page
- Content changes to separation page in Original Claim (OC) flow
- Content changes to BDD supporting evidence page (two states, one for yes, one for no)
- Content changes to BDD evidence upload page
- Content changes to BDD and OC post-submission page

---
## Tasks
- [ ] TBD

## Acceptance Criteria
- [ ] Content changes completed as specified in the following sketch file for the following pages:
- Content changes to BDD intro page
- Content changes to separation page in Original Claim (OC) flow
- Content changes to BDD supporting evidence page (two states, one for yes, one for no)
- Content changes to BDD evidence upload page
- Content changes to BDD and OC post-submission page
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

### PR DESCRIPTION
**Pull request by christinec-fftc**
## Summary
Users who are active service members submitting a 526 under the BDD (Benefits Delivery at Discharge) Program will need to also submit a Part A Self-Assessment form as of April 1 in order to get an exam before discharge from service.

This ticket is to implement design and content changes on the following pages of the form to inform/direct the veteran of the requirement to complete and upload the Part A Self-Assessment form along with the claim to not hold-up further processing:

- Content changes to BDD intro page
- Design and Content changes to BDD supporting evidence page (two states, one for yes, one for no)
- Design changes to BDD evidence upload page
- Design changes to BDD post-submission page

 _(Which team do you work for, does your team own the maintenance of this component?)_
BDEx (Benefits Disability Experience)

## Related issue(s)
[#56570 Implement changes for BDD - SHA Part 1](https://github.com/department-of-veterans-affairs/va.gov-team/issues/56570)

## Testing done
_Describe what the old behavior was prior to the change_
Previously the Separation Health Assessment form was not required, so there was no mention of it

_Describe the steps required to verify your changes are working as expected_
File a new disability claim with a date of release 90 to 180 days from today

_Describe the tests completed and the results_
Manual: Stepped through and visually verified the changes for all 4 views. Ran axe checks, color blindly, zoom to 400%, voiceover on mac. 

Automation
Unit tests added to verify new/modified content and new components. 
Verified e2e tests still working (no new input elements or interactions were added so no modifications to existing tests)

## Screenshots
image
image
image
image
image
image
image
image

## What areas of the site does it impact?
526 Form for Disability Compensation

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by sethdarragile6 — APPROVED** [review]
I pulled this locally and tested the changes. All looks good! Nice work! 🎉

File: src/applications/disability-benefits/all-claims/content/common.jsx [hunk 1]
```diff
@@ -46,3 +47,17 @@ export const ContactCrisis = () => (
...
   </>
 );
+
+export const bddAlertBegin = (
+  <>
+    <p className="vads-u-font-size--base">
+      You’ll need to upload your completed{' '}
+      <a href={DBQ_URL} target="_blank" rel="noreferrer">
+        Separation Health Assessment - Part A Self-Assessment
+      </a>{' '}
```
Comment (Mottie) [inline src/applications/disability-benefits/all-claims/content/common.jsx:57]: Our a11y & content people have started recommending including text to let the user know that the link opens a page in a new tab.

File: src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -113,10 +115,35 @@ class IntroductionPage extends React.Component {
...
+                    . We recommend you download and fill out this form on a
+                    desktop computer or laptop. Then return to this page to
+                    start the application process.
+                  </p>
+                  <p>
+                    <strong>Note:</strong> We estimate that it will take you at
+                    least 30 minutes to complete this form.
+                  </p>
+                  <p>
+                    You'll also have a chance to provide this type of evidence
```
Comment (Mottie) [inline src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx:142]: Not sure why eslint isn't catching these... the single quote (`'`) should be replaced by a typographical apostrophe (`’`)

File: src/applications/disability-benefits/all-claims/content/evidenceTypesBDD.jsx [hunk 3]
```diff
@@ -10,24 +26,50 @@ export const evidenceTypeTitle = (
...
-export const evidenceLayStatements = `Additional VA forms, medical records,
-  separation documents (DD Form 214), supporting (lay) statements, or other
-  evidence`;
+
+export const bddShaOtherEvidence = (
+  <>
+    <strong>
+      Required Separation Health Assessment - Part A Self-Assessment
+    </strong>{' '}
+    or other documents like your DD214, supporting (lay) statements, or other
```
Comment (Mottie) [inline src/applications/disability-benefits/all-claims/content/evidenceTypesBDD.jsx:35]: I don't think an active duty service member (applying for a BDD) would have a DD214 (discharge papers)?

File: src/applications/disability-benefits/all-claims/content/common.jsx (hunk 1 above)
Comment (lydiahooper) [reply src/applications/disability-benefits/all-claims/content/common.jsx:57]: This content was reviewed by OCTO and this is the first I'm hearing of this so we'll keep this in mind for the future.

File: src/applications/disability-benefits/all-claims/content/evidenceTypesBDD.jsx (hunk 3 above)
Comment (lydiahooper) [reply src/applications/disability-benefits/all-claims/content/evidenceTypesBDD.jsx]: Again, this content was reviewed by OCTO so we feel confident we can leave it as-is.

**Review by rsmithadhoc — APPROVED** [review]

**Review by Mottie — APPROVED** [review]
LGTM

**Pull request merged**